tests/integration/distributed_tracing/w3c/test_force_keep_headers_not_sampled_non_newrelic.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Tests that remote_parent_not_sampled = 'always_on' works.
+*/
+
+/*INI
+newrelic.transaction_events.attributes.include=request.uri
+newrelic.distributed_tracing.sampler.remote_parent_not_sampled = 'always_on'
+*/
+
+/*HEADERS
+X-Request-Start=1368811467146000
+Content-Type=text/html
+Accept=text/plain
+User-Agent=Mozilla/5.0
+Referer=http://user:pass@example.com/foo?q=bar#fragment
+traceparent=00-87b1c9a429205b25e5b687d890d4821f-7d3efb1b173fecfa-00
+*/
+
+/*ENVIRONMENT
+REQUEST_METHOD=POST
+CONTENT_LENGTH=348
+*/
+
+/*EXPECT
+ok - insert function succeeded
+ok - traceparent sampled flag ok
+ok - tracestate sampled flag ok
+ok - tracestate priority ok
+*/
+
+require_once(realpath (dirname ( __FILE__ )) . '/../../../include/tap.php');
+
+header('Content-Type: text/html');
+header('Content-Length: 41');
```

Comment:
Why are these needed for testing this functionality? If it's not needed, please remove it.